examples/dynamoathenamigration/migration.go

```diff
@@ -404,6 +404,10 @@ func (t *task) fromS3ToChan(ctx context.Context, dataObj dataObjectInfo, eventsC
 	afterCheckpoint := afterCheckpointIn
 
 	scanner := bufio.NewScanner(gzipReader)
+	// By default maximum size of buffer for scanner is 60KB, which can be too
+	// low for dynamo entries which can be up to 400KB. Let's use 10MB as max.
+	buf := make([]byte, 0, 64*1024)
+	scanner.Buffer(buf, 10*1024*1024)
```

Comment:
10MB sounds pretty big, did you pick this number for a particular reason?

Reply:
on the other hand, looking at decoder.More I am not that convinced to use it. Documentation is rather lacking and I am not sure about edge cases. Do you mind if I just use smaller buffer and keep it without changing to decoder @codingllama ?

Reply:
What edge cases are you concerned about? We could double-check in the Go playground.

Ultimately up to you, but I'd rather avoid a fixed buffer if memory isn't a concern.